build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -62,7 +62,7 @@
   <Target Name="SetupDownloadHostAndSharedFxInputsOutputs" DependsOnTargets="Init">
     <PropertyGroup>
       <CoreSetupBlobRootUrl>https://dotnetcli.azureedge.net/dotnet/</CoreSetupBlobRootUrl>
-      <CoreSetupBlobRootUrlWithChannel>$(CoreSetupBlobRootUrl)$(CoreSetupChannel)</CoreSetupBlobRootUrlWithChannel>
+      <CoreSetupBlobRootUrlWithChannel>$(CoreSetupBlobRootUrl)$(Channel)</CoreSetupBlobRootUrlWithChannel>
```

Comment:
This has to be core-setup channel.  The core-setup channel is different than the CLI channel.  It just so happens that today they are both "master", but we need to keep them separate.